File: app/src/main/java/com/example/communityeventmanagementsystem/domain/model/TrustedApplication.kt
package com.example.communityeventmanagementsystem.domain.model

data class TrustedApplication(
    val id: Long,
    val userId: Long,
    val communityName: String,
    val reason: String?,
    val experience: String? = null,
    val status: String,
    val adminNotes: String?,
    val createdAt: String
)

File: communityApp-Backend/app/Models/TrustedApplication.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class TrustedApplication extends Model
{
    protected $fillable = [
        'user_id',
        'community_name',
        'reason',
        'experience',
        'status',
        'reviewed_by',
        'admin_notes',
        'applied_at',
        'reviewed_at'
    ];

    public function user()
    {
        return $this->belongsTo(
            User::class
        );
    }

    public function reviewer()
    {
        return $this->belongsTo(
            User::class,
            'reviewed_by'
        );
    }
}